packages/common/src/utils/filters.ts

```diff
@@ -187,9 +188,10 @@ export const getFilterRuleWithDefaultValue = <T extends FilterRule>(
                     const valueIsDate =
                         value !== undefined && typeof value !== 'number';
 
+                    // NOTE: Using .format() makes this a standard ISO string
                     const timestampValue = valueIsDate
-                        ? moment(value).format('YYYY-MM-DDTHH:mm:ssZ')
-                        : moment().utc(true).format('YYYY-MM-DDTHH:mm:ssZ');
```

Comment:
This change makes it so that this input is initialized with a local time value instead of a UTC value. This is a change we are making, but it's intentional.